homeassistant/components/fronius/sensor.py

```diff
@@ -150,8 +151,14 @@ async def async_update(self):
         values = {}
         try:
             values = await self._update()
+            self._connection_failure = False  # reset connection failure
```

Comment:
Please only wrap the line that can raise in the try... except block. Use an `else:` block if needed.

Reply:
Done